### PR TITLE
Document current maximum verbosity config value

### PR DESCRIPTION
**Pull request by cosgroveb**
Hey @JelteF, today I was looking to crank the log verbosity up to hopefully diagnose a vexing issue and had to look at the source to learn that setting it to anything higher than 2 would be useless.

Cheers

### CODE REVIEW

**Comment by JelteF** [comment]
I think it's a good idea to document this. But a verbosity of 3 actually has effect too:
```
lib/usual/safeio.c
62: else if (cf_verbose > 2)
78: else if (cf_verbose > 2)
106:    } else if (cf_verbose > 2) {
128:    else if (cf_verbose > 2)
158:    } else if (cf_verbose > 2)
172:    if (res < 0 && (errno != EINPROGRESS || cf_verbose > 2))
176:    else if (cf_verbose > 2)
193:    else if (cf_verbose > 2) {
```

File: doc/config.md [hunk 1]
```diff
@@ -554,6 +554,7 @@ Default: 1
 
 Increase verbosity.  Mirrors the "-v" switch on the command line.
 For example, using "-v -v" on the command line is the same as `verbose=2`.
+2 is the highest currently-supported verbosity.
```
Comment (JelteF) [inline doc/config.md:557]:
```suggestion
3 is the highest currently-supported verbosity.
```

Comment (cosgroveb) [reply doc/config.md]: Thank you for the heads-up! Fixed.

Comment (cosgroveb) [reply doc/config.md]: 2 is currently filling up my disks. I hope that I do not need 3! 😆

**Review by JelteF — APPROVED** [review]

**Pull request merged**